src/shared/psi-util.h

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#pragma once
+
+#include <stdbool.h>
+
+#include "time-util.h"
+
+#define PRESSURE_CPU_FILE "/proc/pressure/cpu"
+#define PRESSURE_IO_FILE  "/proc/pressure/io"
+#define PRESSURE_MEM_FILE "/proc/pressure/memory"
```

Comment:
i am not fan of such defines. The name of the define maps 1:1 to its value, just lowercased and with _ replaced by /, so what's the point?

I think this would be simpler to grok if the strings would just be literally... Extra indirections sound unnecessary and just make code harder to understand.

I am all for defines if the value is kinda random or things shall be configurable, but its not the case here.